lib/utils/normalizers.ts

```diff
@@ -1,16 +1,78 @@
-import * as url from "url";
-import UrlValueParser = require("url-value-parser");
+import * as url from 'url';
+import UrlValueParser = require('url-value-parser');
 
-export function normalizePath(originalUrl: string, extraMasks: Array<RegExp>, placeholder: string) {
+export function normalizePath(
+  originalUrl: string,
+  extraMasks: Array<RegExp>,
+  placeholder: string,
+) {
   const { pathname } = url.parse(originalUrl);
   const urlParser = new UrlValueParser({ extraMasks });
   return urlParser.replacePathValues(pathname, placeholder);
 }
 
 export function normalizeStatusCode(statusCode: number): string {
-  if (statusCode >= 200 && statusCode < 300) return "2XX";
-  if (statusCode >= 300 && statusCode < 400) return "3XX";
-  if (statusCode >= 400 && statusCode < 500) return "4XX";
+  if (statusCode >= 200 && statusCode < 300) return '2XX';
+  if (statusCode >= 300 && statusCode < 400) return '3XX';
+  if (statusCode >= 400 && statusCode < 500) return '4XX';
```

Comment:
I wouldn't normalize it - it has no real value as we can do it in the query to prometheus